### ISSUE TITLE
Training Statistics Documentation

### ISSUE THREAD

**Issue by cdelv** [SciML/NeuralPDE.jl#576]
When training a neural network is helpful to know the statistics of the loss function. In Tensorflow or deepXDE, for example, usually one can plot the loss function vs the epochs. Also, when evaluating the model knowing the value of the loss function in those evaluations is essential. In the documentation, I couldn't find examples of this and don't know how to do it. It would be nice if you could add some examples to the documentation. Also, how to add a progress bar to the training cycle.

**Comment by ChrisRackauckas** [comment]
That's done by the logging functionality, which right now is under-documented because @zoemcc has a major PR to add a whole Tensorboard etc. thing. That should be merged soon?
